# vendor/upstream/tantivy
# upstream commit 72d1ef9a6468aa68bbc69dcc80cdf60aaf64364d
diff --git a/Cargo.toml b/Cargo.toml
index f1f2e51..0fcbbd0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -71,9 +71,6 @@ futures-channel = { version = "0.3.28", optional = true }
 fnv = "1.0.7"
 typetag = "0.2.21"
 
-[target.'cfg(windows)'.dependencies]
-winapi = "0.3.9"
-
 [dev-dependencies]
 binggan = "0.15.3"
 rand = "0.9"

# vendor/upstream/fs4
# upstream commit 511e2d2a2a3a730e1158a6fd72e4acb20f65998a
diff --git a/Cargo.toml b/Cargo.toml
index c14d954..474d7c7 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,10 +27,6 @@ fs-err3-tokio = ["fs-err3", "fs-err3/tokio"]
 [target.'cfg(not(windows))'.dependencies]
 rustix = { version = "1", features = ["fs"] }
 
-[target.'cfg(windows)'.dependencies.windows-sys]
-version = "0.59"
-features = ["Win32_Foundation", "Win32_Storage_FileSystem", "Win32_System_IO"]
-
 [dependencies]
 async-std = { version = "1", optional = true }
 fs-err2 = { package = "fs-err", version = "2", optional = true }
